src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/EndpointResource.cs

```diff
@@ -110,5 +111,17 @@ private Uri BuildUri(ResourceRequestContext context)
             builder.Scheme = Protocol;
             return builder.Uri;
         }
+
+        private static string Getfqdn()
+        {
```

Comment:
Dynamically determine the value is the best, unfortunately I haven't found a good way.

Agree merge it for now to unblock the multi-machine lab runs. We can do the necessary improvement later on.